src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
```

Comment:
The .stars__star class is set to display flex, but it doesn't seem to be necessary as it doesn't have any child elements to control the layout of. If this is unintentional, you can remove it to simplify the code.